user/views.py

```diff
@@ -0,0 +1,18 @@
+from django.views.generic import TemplateView, CreateView
+from django.urls import reverse_lazy
+from .models import User
+from .forms import SignUpForm
+
+
+class HomeView(TemplateView):
+    template_name = "user/home.html"
+
+
+# Create your views here.
+class SignUpView(CreateView):
+    model = User
+    form_class = SignUpForm
+    # The form class to instantiate.
+    success_url = reverse_lazy('user:home')
+    # プロジェクトのURLConf(url.py)が読み込まれる前に、URLの返す
```

Comment:
```suggestion
    # The form class to instantiate.
    form_class = SignUpForm
    # プロジェクトのURLConf(url.py)が読み込まれる前に、URLの返す
    success_url = reverse_lazy('user:home')
```
コメントは対象箇所の上に書くのが一般的な気がします